Funky-Chickens/workFORCEmanagement
# El indivemps

Changed:

Get function for One employee in order to include the access to info through the join tables

We need to display a user's trainings and comptuers on their individual/detail view.

TO TEST:

1. might want to re-roll the db and re-seed it -- make sure you have the latest master and can seed the join tables! (`sequelize db:migrate:undo:all` then `sequelize db:migrate` then `sequelize db:seed:all`
2. run `npm start`
3. go to `localhost:4000/employees`
4. Click on any given employee between 1 and 5 and see if their info displays for both trainings and computer. Employees 6-10 have not been assigned computers yet.

Issue #3
